kernel-modules/BLOCKLIST

```diff
@@ -92,3 +92,6 @@
 # Since kmods are not supported anyway, block these combination
 ~.*\.el8_9\..* * mod
 ~.*\.el9_4\..* ~2\.[3-9]\..* *
+
+# Block all kernels for module 2.10 and newer
+* ~2\.\d{2}\..* *
```

Comment:
Seems blocking them in the scripts didn't work? I don't particularly like us depending on the blocklist for this.

This will not stop the scripts from configuring Falco to build the eBPF probe, which means at best we'll be wasting time configuring and copying files around that we don't need and at worst upstream might break the step and we'll need to fix it for newer versions... That again, we'll not build.

Can we take another crack at blocking this in the scripts? I'm willing to give it a shot myself if needed.

Reply:
I wasn't that happy with how it looked in the scripts, but I'll have another go at it